k8s/jiva/snapshot-cleanup.sh

```diff
@@ -0,0 +1,89 @@
+    #!/bin/bash
+
+    usage()
+    {
+        echo "Usage: bash snapshot-cleanup.sh <pv-name> <number_of_snapshots_to_delete>"
```

Comment:
"bash" here seems to be redundant there is #! in the script, so making it executable should sufficient. Also, how can a user know, that this script is for JIVA and not for cStor?